frontend/national-shooting-championships/index.html

```
@@ -0,0 +1,704 @@
+            <div class="shooting-timeline-card">
+              <span class="shooting-timeline-year">1957</span>
+              <h3>Inaugural National Championships</h3>
+              <p>First Senior National Shooting Championships held in New Delhi. Organised by the National Rifle Association of India (NRAI).</p>
+            </div>
+          </div>
+          <div class="shooting-timeline-step">
+            <div class="shooting-timeline-marker"></div>
+            <div class="shooting-timeline-card">
+              <span class="shooting-timeline-year">1961</span>
+              <h3>NRAI Joins ISSF</h3>
+              <p>National Rifle Association of India affiliates with International Shooting Sport Federation, connecting Indian shooters to the global stage.</            </div>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)